src/API/Site/Controllers/MerchantCenter/ShippingRateController.php

```diff
@@ -304,69 +328,21 @@ protected function get_delete_rate_callback(): callable {
 	 * @return array Array of shipping rates grouped by country code.
 	 */
 	protected function get_all_shipping_rates(): array {
-		$results = $this->create_query()->set_limit( 100 )->get_results();
-		$rates   = $this->group_rates_by_country( $results );
-
-		return array_map( [ $this, 'prepare_country_rates_for_output' ], $rates );
+		return $this->create_query()
+					->set_limit( 200 )
```

Comment:
No reason really. I'm not sure why it was there and there was no comment so I tried not to touch it the first time. But then realized we can have at least as many shipping rates as the number of countries to I increased to more than 100. But you're right there's no logic to it so I've now just removed it so that it won't confuse the next person who looks at it in the future 😅